README_BUILD.md

```diff
@@ -120,6 +120,7 @@ This step also installs some Python dependencies:
 * `Pygments` - For use by the `CodeHilite` extension used by `markdown` in `script_doc.py`. Installed from wheel package in `packages/`.
 * `requests` - Installed using pip
 * `pyaml` - Installed using pip
+* `minio` - Used for interacting with S3-compatable object storage
```

Comment:
Does this means we can replace `boto3`?
Also not that we use a `build_tools/s3.py`, do we need to change/remove that?